ocean/coupling_files_E3SM/create_E3SM_MPAS_coupling_files.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+"""
+This script creates coupling files needed to run an MPAS-Ocean and
+MPAS-seaice within E3SM.
+"""
+import os
+import shutil
+import subprocess
+import ConfigParser
```

Comment:
To support python 3, this would need to change to:
```suggestion
ifrom six.moves.configparser import ConfigParser
```

Reply:
By the way, `six` is a package with an overly cute name (2 x 3 = six) for supporting certain packages in python 2 and 3 at the same time where no other solution exists.